commit ad275d50720fbbaf65bc0668eed0297df5d27cad
Author: Ulrich Sibiller <uli42@gmx.de>
Date:   Thu Dec 3 01:00:59 2020 +0100

    Events.c: fix leftover DEBUG and TEST definitions
    
    causing DEBUG output in regular builds.

diff --git a/nx-X11/programs/Xserver/hw/nxagent/Events.c b/nx-X11/programs/Xserver/hw/nxagent/Events.c
index 2487db69..332cb2f3 100644
--- a/nx-X11/programs/Xserver/hw/nxagent/Events.c
+++ b/nx-X11/programs/Xserver/hw/nxagent/Events.c
@@ -3135,9 +3135,6 @@ int nxagentCheckWindowConfiguration(XConfigureEvent* X)
   return 1;
 }
 
-#define DEBUG
-#define TEST
-
 int nxagentHandleConfigureNotify(XEvent* X)
 {
   #ifdef DEBUG
@@ -3153,7 +3150,7 @@ int nxagentHandleConfigureNotify(XEvent* X)
   fprintf(stderr, "%s: References:\n", __func__);
   fprintf(stderr, "%s:   DefaultWindow[0]: [0x%x]\n", __func__, nxagentDefaultWindows[0]);
   fprintf(stderr, "%s:   DefaultRootWindow(DISPLAY) [0x%lx]\n", __func__, DefaultRootWindow(nxagentDisplay));
-#endif
+  #endif
 
   if (nxagentOption(Rootless) == True)
   {
